src/service/viewer-impl.js

```diff
@@ -273,15 +280,9 @@ export class Viewer {
             // Parent asked for viewer JS. We must be embedded.
             || (this.win.location.search.indexOf('amp_js_v') != -1))
         || this.isWebviewEmbedded_
+        || this.isCctEmbedded_
```

Comment:
This also affects viewer messaging. Need to confirm that you setup the viewer origin, referral url, viewer url and that you actually setup messaging.

Reply:
Yes, I updated the AGSA integration script already and will be rolling it out over the next week. It should be at 100% on Wednesday.